index.html

```diff
@@ -2101,11 +2104,16 @@
           </p>
         </li>
         <li>
-          If the argument <var>options</var> contains invalid values,
+          If the first argument <var>options</var> contains invalid values,
           reject <var>promise</var> with a <code>"<a>SyntaxError</a>"</code>,
           and terminate this algorithm.
         </li>
         <li>
+          If the second argument <var>callback</var> is not an instance of
```

Comment:
You could say: "is not a Function" and link Function to the Terminology (Function is defined in http://heycam.github.io/webidl/#common-Function).